### ISSUE TITLE
Command "GetCleanState" returns "CleanReport" and twice "ChargeState" (T9)

### ISSUE THREAD

**Issue by unclej84** [mrbungle64/node-red-contrib-ecovacs-deebot#9]
Using command "GetCleanState" results in three replies from the robot - once "CleanReport" (expected) and twice "ChargeState".
Can "ChargeState" somehow be filtered out? Is this similar for other models?

**Comment by mrbungle64** [comment]
@unclej84 

> Is this similar for other models?

No, it's not similar for other models, but that's how it is implemented in the library for MQTT/JSON based models.

> once "CleanReport" (expected) and twice "ChargeState"

With the OZMO 920 the `ChargeState` is only emitted once. Can you reproduce it with the T9?

**Comment by unclej84** [comment]
Yes, reproduceable - I tried several times and receive `ChargeState` always twice
